src/main/java/core/basesyntax/converter/DataConverterImpl.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax.converter;
+
+import core.basesyntax.model.FruitTransaction;
+import java.util.ArrayList;
+import java.util.List;
+
+public class DataConverterImpl implements DataConverter {
+    @Override
+    public List<FruitTransaction> covertToTransaction(List<String> data) {
+        List<FruitTransaction> transactions = new ArrayList<>();
+
+        for (int i = 1; i < data.size(); i++) {
+            String[] parts = data.get(i).split(",");
```

Comment:
Consider protecting your code from potential issues by checking if the 'parts' array has the expected length before accessing its elements.